modules/library/main/src/main/java/org/geotools/filter/visitor/ExtractBoundsFilterVisitor.java

```diff
@@ -397,7 +383,79 @@ public Object visit(Subtract expression, Object data) {
 	public Object visitNullFilter(Object data) {
 		return infinity();
 	}
-    
-    
-    
+
+    @Override
+    public Object visit(After after, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(AnyInteracts anyInteracts, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(Before before, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(Begins begins, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(BegunBy begunBy, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(During filter, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(EndedBy endedBy, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(Ends ends, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(Meets meets, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(MetBy metBy, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(OverlappedBy overlappedBy, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(TContains contains, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(TEquals equals, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(TOverlaps contains, Object data) {
+        return infinity();
+    }
+
+    @Override
+    public Object visit(PropertyIsNil filter, Object data) {
+        return infinity();
```

Comment:
yeah...i originally just did the DURING query so that it would behave like a BETWEEN query for temporal predicates but then did it the same for the others like toverlaps, tequals, etc.